### PR TITLE
INS-1969: NodeKeeper tests

### PR DESCRIPTION
**Pull request by krestkrest**
**- What I did**
Added tests for NodeKeeper, simplified internal NodeKeeper states
**- How to verify it**
Run unit and integration tests

### CODE REVIEW

File: network/interfaces.go [hunk 1]
```diff
@@ -220,31 +220,6 @@ type ConsensusInfo interface {
 	IsJoiner() bool
 }
 
-//go:generate minimock -i github.com/insolar/insolar/network.UnsyncList -o ../testutils/network -s _mock.go
```
Comment (AndreyBronin) [inline network/interfaces.go]: remove mock file

File: network/nodenetwork/nodekeeper.go [hunk 2]
```diff
@@ -106,12 +107,12 @@ func createOrigin(configuration configuration.HostNetwork, certificate insolar.C
 	), nil
 }
 
-func resolveAddress(configuration configuration.HostNetwork) (string, error) {
-	addr, err := net.ResolveTCPAddr("tcp", configuration.Transport.Address)
+func resolveAddress(configuration configuration.Transport) (string, error) {
```
Comment (AndreyBronin) [inline network/nodenetwork/nodekeeper.go]: I think we should remove  `resolveAddress ` and `createOrigin ` from NodeNetwork, we can get origin from host network

File: network/interfaces.go (hunk 1 above)
Comment (krestkrest) [reply network/interfaces.go]: removed

File: network/nodenetwork/nodekeeper.go (hunk 2 above)
Comment (krestkrest) [reply network/nodenetwork/nodekeeper.go]: I will create a separate task for this refactoring

**Pull request merged**